src/main/java/codeu/controller/Stopwords.java

```diff
@@ -0,0 +1,871 @@
+      list.add("Swimming");
+      list.add("Swiss");
+      list.add("Swords");
+      list.add("Table");
+      list.add("tabletop");
+      list.add("Taekwondo");
+      list.add("tag");
+      list.add("Tai");
+      list.add("tanning");
+      list.add("Tasting");
+      list.add("Tatting");
+      list.add("Taxidermy");
+      list.add("Tea");
+      list.add("team");
+      list.add("Tennessee");
+      list.add("tennis");
+      list.add("Tesla");
+      list.add("Tetrazzini");
+      list.add("Tetris");
+      list.add("Texas");
+      list.add("Textiles");
+      list.add("Texting");
+      list.add("Thanksgiving");
+      list.add("The");
+      list.add("Theater");
+      list.add("Thirteen");
+      list.add("Thousand");
+      list.add("throwing");
+      list.add("time");
+      list.add("To");
+      list.add("toast");
+      list.add("Toaster");
+      list.add("Tomato");
+      list.add("Tombstone");
+      list.add("Tool");
+      list.add("Tour");
+      list.add("Toy");
+      list.add("Train");
+      list.add("Trainspotting");
+      list.add("Traveling");
+      list.add("Treasure");
+      list.add("Tree");
+      list.add("Trekkie");
+      list.add("Triathlon");
+      list.add("Tuna");
+      list.add("Turducken");
+      list.add("turf");
+      list.add("turkey");
+      list.add("Tutoring");
+      list.add("TV");
+      list.add("Twirling");
+      list.add("Tying");
+      list.add("types");
+      list.add("Ultimate");
+      list.add("unspecified");
+      list.add("Urban");
+      list.add("Utah");
+      list.add("Vehicle");
+      list.add("Vermont");
+      list.add("Vichyssoise");
+      list.add("Victor");
+      list.add("Video");
+      list.add("Videophilia");
+      list.add("Vintage");
+      list.add("Violin");
+      list.add("Virginia");
+      list.add("Volleyball");
+      list.add("Volunteer");
+      list.add("Waffle");
+      list.add("waffles");
+      list.add("Walking");
+      list.add("Warhammer");
+      list.add("Washington");
+      list.add("Watcher");
+      list.add("watching");
+      list.add("Water");
+      list.add("Weather");
+      list.add("Web");
+      list.add("wedges");
+      list.add("Weightlifting");
+      list.add("weights");
+      list.add("West");
+      list.add("Wheeling");
+      list.add("White");
+      list.add("Windsurfing");
+      list.add("Wine");
+      list.add("wing");
+      list.add("Wingsuit");
+      list.add("Wisconsin");
+      list.add("with");
+      list.add("Wood");
+      list.add("Woodworking");
+      list.add("Working");
+      list.add("World");
+      list.add("Worldbuilding");
+      list.add("Wrestling");
+      list.add("writing");
+      list.add("Wyoming");
+      list.add("Wyomingetball");
+      list.add("Yoga");
+      list.add("York");
+      list.add("yourself");
+      list.add("YoYo");
+      list.add("Yo-yoing");
+      list.add("Ziplining");
+      list.add("zucchini");
+      list.add("Zumba");
```

Comment:
Put the words into a text file and read in each word from the file to add to the list. This way, the code is less repetitive and more readable.